hugegraph-dist/src/assembly/descriptor/assembly.xml

```diff
@@ -51,10 +51,6 @@
         <fileSet>
             <directory>${top.level.dir}/hugegraph-dist/release-docs/</directory>
             <outputDirectory>/</outputDirectory>
-            <includes>
-                <include>LICENSE*</include>
-                <include>NOTICE*</include>
-            </includes>
```

Comment:
seems the output dir is not right? 

we should put `LICENSE & NOTICE & DISCLAIMER` into root dir, and then copy `release-docs`'s other license files to the root

Reply:
yes, the licenses, notice, license all the them in the root, include the licenses dir all file